pynance/data/retrieve.py

```diff
@@ -10,14 +10,14 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, start, end = datetime.today().date().isoformat()):
```

Comment:
the args kwargs approach would require changing calling classes wouldn't they? that is investor-bot code.